### PR TITLE
PSMDB-1123 Release notes 4.4.16-16

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/conf.py
new file:   source/release_notes/4.4.16-16.rst
modified:   source/release_notes/index.rst

### CODE REVIEW

**Review by ktrushin — APPROVED** [review]

File: source/release_notes/4.4.16-16.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Find the full list of changes in the `MongoDB 4.4.16 Community Edition release notes <https://www.mongodb.com/docs/v4.4/release-notes/4.4/#4.4.16---aug-19--2022>`_.
+
+Improvements
+================================================================================
+
+* :jirabug:`PSMDB-1046`: Make the ``kmipKeyIdentifier`` option not mandatory
+  
+.. note::
+
+   If you have configured data at rest encryption and wish to upgrade |PSMDB|, go through the `encrypting existing data steps <https://www.mongodb.com/docs/v4.4/tutorial/configure-encryption/#std-label-encrypt-existing-data>`_ during the upgrade as follows:
```
Comment (sandraromanchenko) [inline source/release_notes/4.4.16-16.rst]: This is only true for cases data-at-rest + KMIP feature (which is still technical preview), so I'd re-phase:
If you have configured data at rest encryption with KMIP server*

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: source/release_notes/4.4.16-16.rst (hunk 1 above)
Comment (nastena1606) [reply source/release_notes/4.4.16-16.rst]: Updated

**Review by sandraromanchenko — APPROVED** [review]

**Comment by kwilktx** [comment]
Add the following bugs - important to note hang and blocking events.
MongoDB Inc. often hides bigger bugs by not including them in the release notes-  instead placing them in the overall changelogs for a particular version.
ex. https://www.mongodb.com/docs/v4.4/release-notes/4.4-changelog/#std-label-4.4.16-changelog

https://jira.mongodb.org/browse/SERVER-63243
This bug fix adjusts the functioning of the range-deleter to prevent balancer getting blocked, hung, or ranges being scheduled behind other ranges.

https://jira.mongodb.org/browse/SERVER-67492
Faile dchunk migrations can lead to recipient shard having different config.transactions records between primaries and secondaries - inconsistent data.

https://jira.mongodb.org/browse/SERVER-60958
This fix avoids server hang in chunk migration when a step-down occurs.


To improve readability you could combine these 2 bugs into 1 line -- or even not include them if worried about too much space.

[SERVER-61321](https://jira.mongodb.org/browse/SERVER-61321) - Improved handling of large/NaN (Not a Number) values for text index version.

[SERVER-60607](https://jira.mongodb.org/browse/SERVER-60607) - Improved handling of large/NaN (Not a Number) values for geo index version.

Once those are added and adjusted - Approved.

**Review by kwilktx — CHANGES_REQUESTED** [review]
Once recommendations from previous comments are added and adjusted - Approved.

Thanks,
Kimberly

**Comment by kwilktx** [comment]
Would change the following note

from this:
.. warning::

   The issue with the inconsistent sharding metadata is observed when running the ``movePrimary`` command on the database that has the Feature Compatibility Version (FCV) set to 4.4 or earlier. Affects MongoDB versions 5.0.0 through 5.0.10 and MongoDB 6.0.0. The issue is fixed upstream in versions 5.0.11 and 6.0.1. Percona Server for MongoDB 5.0.11-10 includes the upstream fixes. To learn more about this bug and remediation steps, refer to the `SERVER-68511 <https://jira.mongodb.org/browse/SERVER-68511>`_.

 ==>> to this:

WARNING: Take caution when upgrading from earlier versions of v4.4.x to later versions of 4.4 or on to v5.0. See SERVER-68511 for more details.  `SERVER-68511 <https://jira.mongodb.org/browse/SERVER-68511>`_.

**Pull request merged**